functional_tests/features/statistical_article_page.feature

```diff
@@ -19,3 +19,58 @@ Feature: Statistical Article Page components
         Then the published statistical article page is displayed with the populated data
         And the published statistical article page has the added table
         And the user can expand the footnotes
+
+    Scenario: A CMS user can add a correction to a Statistical Article Page
```

Comment:
How easy would it be to test some of these scenarios in unit tests, instead of in the functional tests?

Given that the functional tests are quite slow to run and are proving a bit flaky/tricky to develop/maintain, I think it would be beneficial to test as much of the logic as we can on the unit side and limit the functional tests to aspects which are better (or only) testable from the browser.

Reply:
Normally unit tests should test a "unit" of code. In this case we would need to setup a page and test the output of a page with a given streamfield, which would turn more into an integration test.

Moreover, a lot of these tests evaluate the code on the admin side, i.e. is it possible to add a correction using the admin panel with JS loaded etc..

We can however later on add unit tests which cover the rendering output (which I purposefully omitted as Jane will be working on the frontend side of things).

Reply:
Poor choice of words on my part, our Python are really a mix of unit and integration tests.

Fair points, I'm just getting concerned about the reliability of our functional tests, which should be addressed separately.